src/components/inputs/validationSchemas.js

```diff
@@ -0,0 +1,15 @@
+export const passwordSchema = {
+  pattern: {
+    value: /^[a-zA-Z0-9!@#$%^&*()_+{}[\]:;'"\\|,<.>/?]{8,}$/,
+    message: "password is weak",
+  },
+  required: "password is required",
+};
+
+export const emailSchema = {
+  pattern: {
+    value: /^[\w-.]+@([\w-]+\.)+[\w-]{2,}$/i,
+    message: "email must be valid.",
+  },
+  required: "email is required",
+};
```

Comment:
add a function here 
getDefaultValidationSchema(type){
return schema based upon type
}

Reply:
Here are the changes:
https://github.com/ahnupsingh/react-crud/pull/18/commits/2a3c6359cbf98f57f1f38db0711da6c2b24fa283
https://github.com/ahnupsingh/react-crud/pull/18/commits/82adb3bed2c59705bf338a86f979ad162595b605

Reply:
@ahnupsingh Please review it.